### PR TITLE
Add segmented control to gallery

### PR DESCRIPTION
**Pull request by nataliesampsell**

### CODE REVIEW

File: examples/flutter_gallery/lib/demo/cupertino/cupertino_segmented_control_demo.dart [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                      ),
+                      const BoxShadow(
+                        offset: const Offset(0.0, 1.0),
+                        blurRadius: 18.0,
+                        spreadRadius: 0.0,
+                        color: _kAmbientShadowOpacity,
+                      ),
+                    ],
+                  ),
+                  child: icons[sharedValue]),
```
Comment (goderbauer) [inline examples/flutter_gallery/lib/demo/cupertino/cupertino_segmented_control_demo.dart]: nit: comma after `]`, and `),` should move to the next line.

**Review by goderbauer — APPROVED** [review]
LGTM once the tree is open again.

**Comment by xster** [comment]
AppVeyor looks like a flake. I restarted it.

**Comment by goderbauer** [comment]
Is this ready to be submitted?

**Comment by xster** [comment]
LGTM

**Pull request merged**